macros/anaUltraScurve.py

```diff
@@ -115,6 +115,8 @@
     myT.Branch( 'scurve_h', scurve_h)
     chi2 = array( 'f', [ 0 ] )
     myT.Branch( 'chi2', chi2, 'chi2/F')
+    ndf = array( 'f', [ 0 ] )
```

Comment:
because I copied chi2, and I figured it didn't matter. TTrees play nicer with floats than ints. I can switch it if you want, I don't really care.